src/Sarif.Viewer.VisualStudio.Core/CodeAnalysisResultManager.cs

```diff
@@ -77,6 +79,7 @@ internal sealed class CodeAnalysisResultManager : IVsSolutionEvents
             this._promptForEmbeddedFileDelegate = promptForEmbeddedFileDelegate ?? this.PromptForEmbeddedFile;
 
             this._allowedDownloadHosts = SdkUIUtilities.GetStoredObject<List<string>>(AllowedDownloadHostsFileName) ?? new List<string>();
+            this._allowedFileExtensions = SdkUIUtilities.GetStoredObject<List<string>>(AllowedFileExtensionsFileName) ?? new List<string>();
```

Comment:
>List [](http://example.com/codeflow?start=125&length=4)

if you change to hashset, you could ignorecase to prevent .pdb and .PDB as two different extensions

Reply:
You need to update the constructor and add the ignorecase option

Reply:
When deserialize the hashset using `SdkUIUtilities.GetStoredObject()`, the comparer will be reset to `EqualityComparer<string>.Default`. I created a override method to deserialize and populate data to existing hashset instance with custom comparer. Please let me know what do you think.